File: src/main/java/net/ndrei/teslacorelib/inventory/ISidedItemHandler.java
package net.ndrei.teslacorelib.inventory;

import net.minecraft.util.EnumFacing;
import net.minecraftforge.items.IItemHandler;

/**
 * Created by CF on 2016-12-14.
 */
public interface ISidedItemHandler extends IItemHandler {
    int[] getSlotsForFace(EnumFacing side);
}

File: src/main/kotlin/net/ndrei/teslacorelib/inventory/ISidedItemHandler.kt
package net.ndrei.teslacorelib.inventory

import net.minecraft.util.EnumFacing
import net.minecraftforge.items.IItemHandler

/**
 * Created by CF on 2017-06-28.
 */
interface ISidedItemHandler : IItemHandler {
    fun getSlotsForFace(side: EnumFacing): IntArray
}
